### PR TITLE
DOCKER-278: java-specific variables were moved to Java layer.

### PR DESCRIPTION
**Pull request by anghelutar**

### CODE REVIEW

File: docker-init-script/src/main/java/eu/xenit/docker/alfresco/InitScriptMain.java [hunk 1]
```diff
@@ -134,24 +134,8 @@ public void process() {
...
 
-        if ("true".equalsIgnoreCase(environment.get("JMX_ENABLED"))) {
-            javaOptions.add("-Dcom.sun.management.jmxremote.authenticate=false");
-            javaOptions.add("-Dcom.sun.management.jmxremote.local.only=false");
-            javaOptions.add("-Dcom.sun.management.jmxremote.ssl=false");
-            javaOptions.add("-Dcom.sun.management.jmxremote");
-            javaOptions.add("-Dcom.sun.management.jmxremote.rmi.port=5000");
-            javaOptions.add("-Dcom.sun.management.jmxremote.port=5000");
-            javaOptions.add("-Djava.rmi.server.hostname=" + environment.get("JMX_RMI_HOST"));
-            globalProperties.put("alfresco.jmx.connector.enabled", "true");
```
Comment (vierbergenlars) [inline docker-init-script/src/main/java/eu/xenit/docker/alfresco/InitScriptMain.java]: I think we will still need this property to be set?

Comment (anghelutar) [reply docker-init-script/src/main/java/eu/xenit/docker/alfresco/InitScriptMain.java]: See slack discussion.
Works fine without setting it.

Comment (vierbergenlars) [reply docker-init-script/src/main/java/eu/xenit/docker/alfresco/InitScriptMain.java]: > Toon Geens 12:00
> I'm still wondering what the alfresco.jmx.connector.enabled option does
> see AlfrescoRmiRegistryFactoryBean
> that's just Alfresco/Spring itself managing JMX in code ?
> 
> Roxana Angheluta 13:36
> I just tried with a container having alfresco.jmx.connector.enabled=false and connecting via jmx works fine on port 5000 without authentication

Comment (vierbergenlars) [reply docker-init-script/src/main/java/eu/xenit/docker/alfresco/InitScriptMain.java]: Okay then

File: 1skeleton/build.gradle [hunk 2]
```diff
@@ -69,6 +70,8 @@ subprojects {
 
         if (project.hasProperty('extraBuildArgs'))
             buildArgs << project.extraBuildArgs
+
+	println buildArgs
```
Comment (vierbergenlars) [inline 1skeleton/build.gradle]: Remove println from the build.gradle

**Pull request merged**